internal/db/nosql/cassandra/cassandra.go

```diff
@@ -266,18 +267,24 @@ func (c *client) Query(stmt string, names []string) *Queryx {
 	return gocqlx.Query(c.session.Query(stmt), names)
 }
 
+func clientError(err error, keys ...string) error {
+	switch err {
+	case ErrNotFound:
+		return errors.Wrap(err, errors.ErrCassandraNotFound(keys...).Error())
+	case ErrUnavailable:
+		return errors.Wrap(err, errors.ErrCassandraUnavailable().Error())
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
It is better to embed failed keys to error message. it is useful for detecting which keys failed.